### ISSUE TITLE
[WOW-API] Addon problem

### ISSUE THREAD

**Issue by donniDevito** [dalaranwow/dalaran-wow#5874]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
We are using epgp as out loot distribution system, and it stores it's values in officer notes. However we noticed that the call to GuildRosterSetOfficerNote when updating values of all members of the guild crushes the game client. To combat it we manually throttle onUpdate and now it takes forever. 
**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1.  Get epgp addon
2. Mass award EP
3. Client crushes.

Alternatively running something like /script for i=1, GetNumGuildMembers() do GuildRosterSetOfficerNote(i, "hello") end
for a middle sized guild(around 300 members total) should do the trick. 
**Include proofs for this behaviour**
It just happens. 

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by MalkaviaDW** [comment]
Hello, 

We have a pending update that should cover this.

Cheers!

**Comment by donniDevito** [comment]
Could you please mark this issue when it's live?
Thanks in advance.
